### PR TITLE
John conroy/remove origin sample refs

### PR DESCRIPTION
**Pull request by yuanzhou**
Following merged PR #704 for TEST

### CODE REVIEW

**Review by AlanSimmons — APPROVED** [review]

**Comment by yuanzhou** [comment]
@AlanSimmons sorry that I must have requested your review by accident. Normally once the PR to DEV approved, I would not need to get the same changes reviewed again.

**Pull request merged**
